pkg/adaptation/result.go

```diff
@@ -499,6 +503,42 @@ func (r *result) adjustSysctl(sysctl map[string]string, plugin string) error {
 	return nil
 }
 
+func (r *result) adjustRdt(rdt *LinuxRdt, plugin string) error {
+	if r == nil {
+		return nil
+	}
+
+	r.initAdjustRdt()
+
+	id := r.request.create.Container.Id
+
+	if rdt.GetRemove() {
+		r.owners.ClearRdt(id, plugin)
+		r.reply.adjust.Linux.Rdt = &LinuxRdt{}
```

Comment:
Does `Remove` need to be set on this as well in order for it to be removed from the OCI spec?

Reply:
Thanks for sharp-eyed review @chrishenzie. Indeed, it needs to be set so that if the original oci spec (before any nri plugins kicked in) contains RDT configuration, the final apply/adjust will then remove it. Fixed.